doc/source/whatsnew/v0.22.0.txt

```diff
@@ -193,6 +193,7 @@ Other API Changes
 - Rearranged the order of keyword arguments in :func:`read_excel()` to align with :func:`read_csv()` (:issue:`16672`)
 - :func:`pandas.merge` now raises a ``ValueError`` when trying to merge on incompatible data types (:issue:`9780`)
 - :func:`wide_to_long` previously kept numeric-like suffixes as ``object`` dtype. Now they are cast to numeric if possible (:issue:`17627`)
+- comment arg is exposed as a named parameter in :func:`read_excel`
```

Comment:
reverse this statement, ``in :func:`read_excel`, the ``comment`` argument is now exposed as a named parameter`` (and need double-backticks)